### ISSUE TITLE
🔤 Updates to the accessibility statement

### ISSUE THREAD

**Issue by NienkeBr** [GCTC-NTGC/gc-digital-talent#15377]
### 🔤 Copy

We want to make some updates to the accessibility statement, including to say that we run all code through automated testing and fix any flagged issues before we release it.

### 🙋‍♀️ Proposed Solution

see A/C

Document with tracked changes: [Accessibility statement_update December 2025.docx](https://github.com/user-attachments/files/24245343/Accessibility.statement_update.December.2025.docx)

#### English:
- [ ] Replace the header "Making our products accessible and usable by all" to "Making our products accessible and usable for everyone"
- [ ] Update the bullet points in that section to:

> - our designers pay attention to: 
>   - designing accessible layouts and interactive elements 
>   - improving colour accessibility 
>   - building accessible features and tools, such as dark mode 
>   - applying findings from user experience research 
> - our developers pay attention to: 
>   - meeting [WCAG 2.1](https://www.w3.org/TR/WCAG21/) requirements 
>   - meeting [Authoring Tools Accessibility Guidelines (ATAG) 2.0](https://www.w3.org/TR/ATAG20/) requirements, including: 
>     - ensuring tools for publishing content on the platform are accessible 
>     - adding safeguards to ensure published content is accessible 
>   - conducting and reviewing automated accessibility testing, including: 
>     - testing all new code with axe-core, a tool that flags common issues such as missing headings and alternative text for images
>     - fixing all issues flagged by automatic testing before deploying to the platform 
>   - reviewing user acceptance test results

<details><summary>Screenshot</summary>
<img width="823" height="370" alt="Image" src="https://github.com/user-attachments/assets/0dc9e4bf-0f57-49b8-94dd-b4065db0fec3" />
</details>

- [ ] Please remove "Before we consider anything to be ready for release, w" and start the sentence as "We work with Fable Tech Labs..." instead.
- [ ] Remove the colon and line break and update the sentence to "..., we encourage you to contact us at support-soutien@talent.canada.ca."
- [ ] Update the sentence to "We reply within two business days."

<details><summary>Screenshot</summary>
<img width="818" height="320" alt="Image" src="https://github.com/user-attachments/assets/17a232e3-c0f6-4dc3-bfd7-89b0e922775a" />
</details>

- [ ] Please add the acronym (CHRC) after the first mention of the Canadian Human Rights Commission: "The Accessibility Commissioner is a member of the Canadian Human Rights Commission (CHRC)."
- [ ] Please remove the bullet point starting with "The Canadian Radio-television and Telecommunications Commission..."
- [ ] Please remove the CHRC acronym from the bullet point
- [ ] Please remove the bullet point starting with "The Canadian Transport Agency..." 
- [ ] Remove the periods at the end of the bulleted list
- [ ] Update the sentence before the CHRC contact details to "If you're not happy with how we respond to your complaint, contact the CHRC:"
- [ ] Remove the hours from the bulleted list and place it underneath.
- [ ] Update the final sentence to "This statement was prepared on November 8, 2022, and updated on December 17, 2025."

<details><summary>Screenshot</summary>
<img width="632" height="394" alt="Image" src="https://github.com/user-attachments/assets/2957c041-032e-4f2d-ba7e-7ddb02fe3ce9" />
</details>

#### French
- [ ] In the sentence under the header "Nos normes", please remove "(GC)"

<details><summary>screenshot</summary>

<img width="1230" height="404" alt="Image" src="https://github.com/user-attachments/assets/dbdc4a7a-0d71-46b9-a959-8f2e03d38f77" />
</details>

- [ ] Update the bullet points in the "Rendre nos produits accessibles et utilisables par tous" section to:

> - Notre équipe de conception prête une attention particulière aux éléments suivants : 
>   - la conception de mises en page et d’éléments interactifs accessibles; 
>   - l’amélioration de l'accessibilité des couleurs; 
>   - la création des fonctionnalités et des outils accessibles, comme le mode sombre; 
>   - l’optimisation l’exploitation des résultats de la recherche en matière de l'expérience utilisateur. 
> - Notre équipe de développement prête une attention particulière aux éléments suivants : 
>   - la conformité aux règles d’accessibilité WCAG 2.1; 
>   - la conformité aux règles d’accessibilité pour les outils d’édition (ATAG) 2.0, notamment :  
>     - l'accessibilité des outils de publication de contenu sur la plateforme;
>     - l'ajout de mesures de protection visant à assurer l'accessibilité du contenu publié.
>   - la réalisation et l’examen des évaluations automatisées d’accessibilité, notamment : 
>     - les tests de tout nouveau code à l’aide d’axe-core, un outil qui signale des problèmes courants comme l’absence de titres ou de texte de remplacement pour les images;
>     - la correction de tous les problèmes signalés par les évaluations automatisées avant le déploiement sur la plateforme.
>   - l'examen des résultats d’évaluation d’acceptation par les utilisateurs.

<details><summary>screenshot</summary>

<img width="877" height="380" alt="Image" src="https://github.com/user-attachments/assets/496229a5-8bb5-4d54-80df-c3c6bc2f4a36" />
</details>

- [ ] Please remove "Avant que nous considérons qu'on produit est prêt à être lancé, n" and start the sentence as "Nous travaillons avec..." instead.
- [ ] Remove the colon and line break for the support email
- [ ] Please update the sentence to "Nous répondons aux demandes de renseignements dans un délai de deux jours ouvrables." 

<details><summary>screenshot</summary>

<img width="880" height="400" alt="Image" src="https://github.com/user-attachments/assets/446e1695-cb69-452c-a734-a097c8e579d4" />
</details>

- [ ] Please add the acronym (CCDP) after the first mention of the Commission canadienne des droits de la personne: "Le commissaire à l’accessibilité est membre de la Commission canadienne des droits de la personne (CCDP)."
- [ ] Please remove the bullet point starting with "Le Conseil de la radiodiffusion ...."
- [ ] Please remove the CCDP acronym from the bullet point
- [ ] Please remove the bullet point starting with "L'Office des transports..."

<details><summary>screenshot</summary>

<img width="896" height="399" alt="Image" src="https://github.com/user-attachments/assets/bd273825-0a26-4df9-9e41-7b5ff45ff2ee" />
</details>

- [ ] Remove the hours from the bulleted list and place it underneath.
- [ ] Update the sentence before the CHRC contact details to "Si vous n’êtes pas satisfait de notre réponse à votre plainte, veuillez contacter la CCDP :"
- [ ] Update the final sentence to "Le présent énoncé a été préparé le 8 novembre 2022 et mise à jour le 17 décembre 2025."

<details><summary>screenshot</summary>

<img width="902" height="346" alt="Image" src="https://github.com/user-attachments/assets/40bfebb1-3cd6-4407-8e54-6c6ba69ba9ae" />
</details>
